example/package.json

```diff
@@ -14,9 +14,16 @@
   },
   "dependencies": {
     "react": "18.2.0",
-    "react-native": "0.71.4"
+    "react-native": "0.71.4",
+    "react-native-reanimated": "^3.0.2"
```

Comment:
Can you explain why we need this when we already have it in our lib `rn-native-component`

Reply:
Need react-native-reanimated in example for component to work